test/integration/targets/apt/tasks/repo.yml

```diff
@@ -148,6 +148,14 @@
         Pin-Priority: 1000
       dest: /etc/apt/preferences.d/foo
 
+  - name: Pin foo=1.0.0
+    copy:
+      content: |-
+        Package: foo
+        Pin: version 1.0.0
+        Pin-Priority: 1002
+      dest: /etc/apt/preferences.d/foo
+
```

Comment:
Shouldn't this be done by modifying the previous task (which also copies data to the same destination file) instead?